src/renderer/ScreenController.tsx

```diff
@@ -135,10 +135,16 @@ export default class ScreenController extends Component {
       this.changeScreen(Screens.Welcome)
     }
 
-    await BackendRemote.rpc.startIo(accountId)
+    if (!dontStartIo) {
+      await BackendRemote.rpc.startIo(accountId)
+    }
     runtime.setDesktopSetting('lastAccount', accountId)
-    log.info('system_info', await BackendRemote.rpc.getSystemInfo())
-    log.info('account_info', await BackendRemote.rpc.getInfo(accountId))
+    BackendRemote.rpc.getSystemInfo().then(info => {
+      log.info('system_info', info)
+    })
+    BackendRemote.rpc.getInfo(accountId).then(info => {
+      log.info('account_info', info)
+    })
```

Comment:
what's the reason for this change?
make account switching faster by not waiting for the result? or did the calls fail?

Reply:
acording to commit message I assume the first.

Reply:
It's both